utils/testrunner/TestRunner.js

```diff
@@ -348,14 +348,21 @@ class TestRunner extends EventEmitter {
   async run() {
     const runnableTests = this._runnableTests();
     this.emit(TestRunner.Events.Started, runnableTests);
-    const pass = new TestPass(this, this._rootSuite, runnableTests, this._parallel, this._breakOnFailure);
-    const termination = await pass.run();
+    this._runningPass = new TestPass(this, this._rootSuite, runnableTests, this._parallel, this._breakOnFailure);
+    const termination = await this._runningPass.run();
+    this._runningPass = null;
     if (termination)
       this.emit(TestRunner.Events.Terminated, termination.message, termination.error);
     else
       this.emit(TestRunner.Events.Finished);
   }
 
+  terminate() {
```

Comment:
I'm adding this here as a testRunner API. I'll probably move process signal handling to the Reporter that will use this api.